includes/Block/Selected_Stories_Block.php

```diff
@@ -70,7 +70,7 @@ class Selected_Stories_Block extends Latest_Stories_Block {
 	 */
 	public function init() {
 		$this->register_script( self::SCRIPT_HANDLE, [ self::STORY_PLAYER_HANDLE, Tracking::SCRIPT_HANDLE ] );
-		$this->register_style( self::SCRIPT_HANDLE, [ self::STORY_PLAYER_HANDLE ] );
+		$this->register_style( parent::SCRIPT_HANDLE, [ self::STORY_PLAYER_HANDLE ] );
```

Comment:
shouldn't this be `::STYLE_HANDLE`?

Reply:
@divyarajmasani 
```	/**
	 * Script handle.
	 *
	 * @var string
	 */
	const SCRIPT_HANDLE = 'latest-web-stories-block';

	/**
	 * Style handle.
	 *
	 * @var string
	 */
	const STYLE_HANDLE = 'latest-stories-block-style';
```

We have to enqueue SCRIPT_HANDLE in admin panel/block editor and STYLE_HANDLE in WP frontend. We don't have any compiled `latest-stories-block-style.css` file, we only have `latest-web-stories-block.css` in our assets/css directory.